packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -147,6 +275,27 @@ export async function getStorageAt<ReturnFormat extends DataFormat>(
 	return format({ eth: 'bytes' }, response, returnFormat);
 }
 
+/**
+ * Get the code at a specific address.
+ *
+ * @param web3Context ({@link Web3Context}) Web3 configuration object that contains things such as the provider, request manager, wallet, etc.
+ * @param address The address to get the code from.
+ * @param blockNumber ({@link BlockNumberOrTag} defaults to {@link Web3Eth.defaultBlock}) Specifies what block to use as the current state for the code query.
+ * @param returnFormat ({@link DataFormat} defaults to {@link DEFAULT_RETURN_FORMAT}) Specifies how the return data should be formatted.
+ * @returns The [data](https://ethereum.org/en/developers/docs/transactions/#the-data-field) at the provided `address`.
+ *
+ * ```ts
+ * web3.eth.getCode("0x033456732123ffff2342342dd12342434324234234fd234fd23fd4f23d4234").then(console.log);
```

Comment:
Mostly functions in Web3Eth class are only returning results of wrappers by passing context, so Could you check if possible, can we use inherit doc feature https://typedoc.org/tags/inheritDoc/ for copying doc from Web3Eth class functions into these wrappers in rendered doc? and add additional @ param  web3Context in these functions. 

This will help us for avoiding duplicate effort for maintaining doc.

Reply:
This would definitely be an improvement over the duplicated docs, however, I cannot get `@inheritdoc` to work. Additionally, I don't think the code example would be inherited based off what's listed here:

![image](https://user-images.githubusercontent.com/20375610/181594095-a9a634a6-857a-4e7a-b375-d804cc8ceb02.png)

Reply:
Made an [issue](https://github.com/ChainSafe/web3.js/issues/5295) to track this

Reply:
Looks like [@remarks](https://typedoc.org/tags/remarks/) should maybe solve the code example being copied